src/MOI_wrapper.jl

```diff
@@ -16,7 +16,8 @@ struct Solution
     objective_constant::Float64
     solve_time::Float64
 end
-const OPTIMIZE_NOT_CALLED = -1
+# The values used by ECOS are from -7 to 10 so -10 should be safe
+const OPTIMIZE_NOT_CALLED = -10
```

Comment:
Maybe a large random negitive number, like `-82381357`, would be safer, incase they expand?

Reply:
Or `nothing` might be even better

Reply:
That would raise an error at this line: https://github.com/jump-dev/ECOS.jl/blob/1e8e25c24c367d18f6ca184fe5046618767ab9f3/src/MOI_wrapper.jl#L21

(`nothing` cannot be converted to `Int`).

Any large negative integer should do the trick.

Reply:
I have changed the struct to allow `nothing`, I don't feel safe using magic numbers ^^